chain/default_syncer.go

```diff
@@ -83,12 +83,13 @@ type DefaultSyncer struct {
 	badTipSets *badTipSetCache
 	consensus  consensus.Protocol
 	chainStore syncerChainReader
+	blockTime  time.Duration
```

Comment:
I am not sure its the chain syncers job to enforce a soft-block time, I could imagine this field going on the consensus interface as well. cc @ZenGround0 you have any thoughts on this?

Reply:
I don't think it should go deeper. But rather than block time, this would be better as an abstract validator.

Reply:
Will replace with an interface (`BlockSemanticValidator`) in the `consensus/` package. Validation refactor (that @ZenGround0 describes) can happen in a follow on.